main.py

```diff
@@ -69,23 +86,91 @@ def draw() -> Shape:
     return target_shape
 
 
+def draw_second_stage() -> Shape:
+    """Draw the next round screen and return the chosen shape"""
+    pygame.draw.rect(SCREEN, (0, 0, 0), (0, 0, 800, 600))
+
+    random.shuffle(shapes)
+    for index, shape in enumerate(shapes):
+        if index > 0:
+            break
+
+        position = shape.map(positions[index])
+
+        # TODO: Fix the issue with pygame and being able to draw the shape from the shape class
+        # TODO: This is a dirty hack because of the 24 hour deadline from start to finish
+        if shape.method == 'rect':
+            pygame.draw.rect(SCREEN, shape.colour, position)
+        elif shape.method == 'circle':
+            pygame.draw.circle(SCREEN, shape.colour, position, shape.radius)
+        elif shape.method == 'polygon':
+            pygame.draw.polygon(SCREEN, shape.colour, position)
+        elif shape.method == 'line':
+            pygame.draw.line(SCREEN, shape.colour, position[0], position[1], shape.radius)
+
+    target_shape = shapes[random.randint(0, 3)]
+    while target_shape.method == last_shape:
+        target_shape = shapes[random.randint(0, 3)]
+
+    font = pygame.font.Font(None, 48)
+    text = font.render(f"{current_round}. Please select the {target_shape.type}", True, (255, 255, 255))
+    text_rect = text.get_rect(center=(400, 50))
+    SCREEN.blit(text, text_rect)
+
+    return target_shape
+
+
+def draw_third_stage() -> Shape:
+    """Draw the next round screen and return the chosen shape"""
+    pygame.draw.rect(SCREEN, (0, 0, 0), (0, 0, 800, 600))
```

Comment:
I've noticed we're doing this a lot, my fault really. I think we'd be good to create a `def clear_screen() -> None:` method to do this. We could even pass a colour tuple, default to black. This will help if we ever wanted to change the screen resolution for example - we only need to update it in one place.